README.md

```diff
@@ -22,20 +22,20 @@ This client supports the following Google Cloud Platform services at a [General
 This client supports the following Google Cloud Platform services at a [Beta](#versioning) quality level:
 
 * [BigQuery](#bigquery-beta) (Beta)
+* [Stackdriver Debugger](#stackdriver-debugger-beta) (Beta)
+* [Stackdriver Error Reporting](#stackdriv-error-reporting-beta) (Beta)
```

Comment:
The anchor seems misspelled. Is it correct?

Reply:
Good catch